doc/sscsm_security.md

```diff
@@ -0,0 +1,76 @@
+# SSCSM security
+
+
+## Threat model
+
+* SSCSM scripts come from the server (potential malicious actor). We are the client.
+* Authenticity of server is not given (our networking is not secure). So we have
+  to expect anyone who can send us UDP packets to the appropriate IP address to be
+  able to act on behalf of the server.
+* The server may not tamper with, or get access to information of, anything besides
+  the stuff explicitly made accessible via the modding API (i.e. gameplay relevant
+  stuff, like map, node definitions, ...).
+  In particular, this excludes for (non-exhaustive) example files, file paths,
+  and settings.
+* DOS is not an issue (as it is already easily possible to DOS a client).
```

Comment:
(it's also low risk (and players quickly learn not to connect to malicious servers); and pulling off the packet spoofing stunt is not easy in general)

Reply:
>pulling off the packet spoofing stunt is not easy in general

I have a bit of a knowledge gap here.

Hence, I preferred being pessimistic. Also, in practice this just *reduces* the probability of the packet sender being malicious.